src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -574,15 +574,17 @@ internal string GetName()
                 // revert thread impersonation for the duration of the call to get the name.
                 RunImpersonated(SafeAccessTokenHandle.InvalidHandle, delegate
                 {
-                    NTAccount ntAccount = User.Translate(typeof(NTAccount)) as NTAccount;
-                    _name = ntAccount.ToString();
+                    NTAccount? ntAccount = User!.Translate(typeof(NTAccount)) as NTAccount;
+                    _name = ntAccount!.ToString();
```

Comment:
This is a strange construction.  Doesn't seem worth changing it to improve the code, but, yeah.